content/blog/2020-03-03-commcall-mar2020.md/2020-03-03-commcall-mar2020.md

```diff
@@ -0,0 +1,89 @@
+
+#### Resources
+- [Taking over maintenance of a software package](https://ropensci.org/blog/2019/06/12/taking-over-maint/), by Scott Chamberlain, Maëlle Salmon & Noam Ross
+- [Relaunching the qualtRics package](https://ropensci.org/blog/2019/04/30/qualtrics-relaunch/), by Julia Silge
+- [Package evolution - changing stuff in your package](https://devguide.ropensci.org/evolution.html), Chapter 14 in  the rOpenSci
+- [The mail must get through](http://www.catb.org/~esr/writings/cathedral-bazaar/cathedral-bazaar/ar01s02.html), by Eric Steven Raymond]
+
+### Speakers {#speakers}
+
+{{< figure src = "julia-silge.jpg" width = "200" alt = "Julia Silge" >}}
+
+_Julia on [GitHub](https://github.com/juliasilge), [Twitter](https://twitter.com/juliasilge), [Website](https://juliasilge.com/), [rOpenSci](https://ropensci.org/authors/julia-silge/)_  
+
+
+
+{{< figure src = "scott-chamberlain.jpg" width = "200" alt = "Scott Chamberlain" >}}
+
+_**Scott Chamberlain** is Co-founder and Technical Lead of rOpenSci. He maintains probably too many packages. Part of Scott’s work involves finding new maintainers for rOpenSci peer-reviewed packages when current maintainers need to move on._
+
+_Scott on [GitHub](https://github.com/sckott/), [Twitter](https://twitter.com/sckottie), [Website](https://scottchamberlain.info/), [rOpenSci](https://ropensci.org/authors/scott-chamberlain/)_  
+
+
+
+{{< figure src = "leonardo-collado-torres.jpg" width = "200" alt = "Leonardo Collado-Torres" >}}
+
+_Leo on [GitHub](https://github.com/lcolladotor), [Twitter](https://twitter.com/fellgernon), [Website](http://lcolladotor.github.io/), [rOpenSci](https://ropensci.org/authors/leonardo-collado-torres/)_  
+
+
+
+{{< figure src = "erin-grand.jpg" width = "200" alt = "Erin Grand" >}}
+
+_Erin on [GitHub](https://github.com/eringrand), [Twitter](https://www.twitter.com/astroeringrand), [Website](http://eringrand.github.io/),  [rOpenSci](https://ropensci.org/authors/erin-grand/)_  
+
+
+
+
+{{< figure src = "elin-waring.jpg" width = "200" alt = "Elin Waring" >}}
+_*Elin Waring is* a Professor of Sociology and Interim Dean of the School of Health Sciences, Human Services and Nursing at Lehman College, CUNY. She teaches research methods and statistics. Elin was part of the unconf17 group that developed the skimr package. skimr has become very popular (257k downloads) and Elin works with Michael Quinn to maintain skimr as they have shepherded it through two major releases.  She formerly was a contributor and maintainer for the Joomla! CMS project, and her approach to maintaining is influenced by that. This includes understanding the importance of having a clear concept of what you are trying to achieve, being able to politely but firmly say no and knowing having users changes everything.
+
+E_li_n on [GitHub](https://github.com/elinw), [Twitter](https://twitter.com/ElinWaring), [Website](https://elinwaring.org/), [rOpenSci](https://ropensci.org/authors/elin-waring/)_
```

Comment:
@steffilazerte Did you use RStudio to edit this file? I'm asking because I see misplaced `
_`. It happened to me countless times to have strange edits happening when editing a file containing emojis from within RStudio. I've never taken the time to explore it too much, I just try to avoid editing such files in RStudio these days. :woman_shrugging:

Reply:
Yup, that would explain it! I was wondering about that. Glad to know it's an emoji thing, I was starting to wonder about other work...

Reply:
Oh hey maybe you and I should just update RStudio! https://github.com/rstudio/rstudio/issues/1701

Reply:
Hah! Awesome thanks I'll give it a try. I've always meant to give Atom a go, but I guess I can stick with Rstudio ;)